### PR TITLE
[2.11.x] DDF-3378 upgraded maven resource plugin

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
Upgrades the maven resource plugin to address a random corruption that occurs when filtering resources in ddf-common. Updated ddf and ddf.bat scripts to get them ready for restart. Will help with the corresponding Alliance PR.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@emanns95 @tbatie 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@clockard
@coyotesqrl
@lessarderic
#### How should this be tested? (List steps with links to updated documentation)
Simply build the distro and test that ddf can still be started on both Linux and Windows.
#### What are the relevant tickets?
[DDF-3378](https://codice.atlassian.net/browse/3378)
[DDF-3346](https://codice.atlassian.net/browse/3346)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by paouelle** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -13,1025 +13,1025 @@
  **/
 -->
```
Comment (lessarderic) [inline pom.xml]: ❓ Can we switch back to 4-space indents and avoid the formatting changes? Pretty sure all of our pom (and most likely xml) files use 4-space indents.

Comment (paouelle) [reply pom.xml]: This is caused by the google format profile :-( It formats everything not just Java

**Comment by paouelle** [comment]
build now

File: pom.xml (hunk 1 above)
Comment (tbatie) [reply pom.xml]: I agree that it would be nice not to include this change for now.

Comment (paouelle) [reply pom.xml]: I agree too. Didn't find an easy way in IntelliJ to address this. If I don't use the goggle profile, the other formatter ends up changing the files and then when the build runs on the command line, some lines gets corrupted sometimes.

**Comment by paouelle** [comment]
build now

**Comment by emanns95** [comment]
Confirmed ddf could be started on both Linux and Windows

**Comment by lessarderic** [comment]
Looks like the latest build was successful even though the pipeline reported a failure.

Since this has been built and tested manually on both, Linux and Windows, merging this PR.

**Pull request merged**